### DISCUSSION TITLE
Copying entire Tree

### DISCUSSION THREAD

**Discussion by Vvoronwe** [TriliumNext/Trilium#4593] [Q&A]
Hello all!

I'm using Trilium for world building purposes, and have run into a particularly thorny issue. I've created a rather detailed tree to act as a broad template for various parts of the world building project with templates and such placed inside the various notes. I was hoping to be able to copy the entire tree itself, but I only seem able to clone it which isn't ideal.

![image](https://github.com/zadam/trilium/assets/157245310/80b6a4cb-f6fd-42c2-ad81-0af047255191)

The tree here is meant to be the baseline from which I can copy/paste for each nation/territory/planet in the project. Is there another method to copy the tree and its contents? Or am I needing to go in and duplicate each note independently?

Thanks all

**Comment by rodrigoalexandrecosta** [comment]
Yeah, cloning definitely is not what you want here. I believe that an easier process than manually duplicate each note would be export the tree, and then re-import it, but I didn't test it and don't know if it will be enough.

edit: just found out that in the tree actions - when you click any note with the secondary/right mouse button - you have an option called "Duplicate subtree" which is exactly what you want.

**Comment by Vvoronwe** [comment]
Oh! That's exactly what I needed, thank you so much!!
